src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `<:OrderedFactor(2)` or `<:Multiclass(2)`; check the scitype
+  with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `method::Symbol=:gevd`: The solver, one of `:gevd` or `:whiten` methods.
+- `cov_w::CovarianceEstimator`=SimpleCovariance: An estimator for the within-class
+    covariance (used in computing within-class scatter matrix, Sw), by default set
+    to the standard `MultivariateStats.SimpleCovariance()` but
+    could be set to any robust estimator from `CovarianceEstimation.jl`.
+- `cov_b::CovarianceEstimator`=SimpleCovariance: The same as `cov_w` but for the between-class
+    covariance (used in computing between-class scatter matrix, Sb).
+- `out_dim::Int=0`: The output dimension, i.e dimension of the transformed space,
+    automatically set if 0 is given (default).
+- `regcoef::Float64=1e-6`: The regularization coefficient (default value 1e-6). A positive
+    value `regcoef * eigmax(Sw)` where `Sw` is the within-class scatter matrix, is added
+    to the diagonal of Sw to improve numerical stability. This can be useful if using
+    the standard covariance estimator.
+- `dist::SemiMetric=SqEuclidean`: The distance metric to use when performing classification
+    (to compare the distance between a new point and centroids in the transformed space),
+    an alternative choice can be the `CosineDist`.Defaults to `SqEuclidean`.
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimentional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projected_class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `projection_matrix`: The matrix used to project `X` into a lower dimensional space.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `classes`: The classes seen during model fitting.
+- `out_dim`: The dimensions the model is projected to.
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `mean`: The mean vector.
+- `class_weights`: The weights of each class.
+- `Sb`: The between class scatter matrix.
+- `Sw`: The within class scatter matrix.
+- `nc`: The number of classes.
+
+# Examples
+
+```
+using MLJ
+
+LDA = @load LDA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = LDA()
+mach = machine(model, X, y) |> fit!
+
+Xproj = transform(mach, X)
+y_hat = predict(mach, x)
```

Comment:
Bug here. i think you mean `X` instead of `x`. Please fix and confirm the edited code runs.